packages/components/src/components/popover/_popover.scss

```diff
@@ -0,0 +1,233 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/scss/vars';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+
+/// Popover component
+/// @access private
+/// @group popover
+@mixin popover {
+  $popover-text-color: $text-01;
+  $popover-caret-offset: 1rem;
+  $popover-box-shadow: 2px 2px 6px 0 rgba(0, 0, 0, 0.2);
+  $popover-offset: 8px;
+
+  .#{$prefix}--popover {
+    position: absolute;
+    z-index: z('floating');
+    display: none;
+  }
+
+  .#{$prefix}--popover--relative {
+    position: relative;
+  }
+
+  .#{$prefix}--popover--open {
+    display: block;
+  }
+
+  .#{$prefix}--popover-contents {
+    @include box-shadow;
+
+    position: relative;
+    width: max-content;
```

Comment:
we've been avoiding this but have we already dropped support for IE?

Reply:
@emyarod the `Popover` work is targetting post-IE11 (v11) stuff 👀